locale/templates/main.ftl

```diff
@@ -344,6 +344,17 @@ plan-details-header = Product details
 plan-details-show-button = Show details
 plan-details-hide-button = Hide details
 plan-details-total-label = Total
+plan-details-list-price = List Price
+
+## coupons
```

Comment:
@IvoJP, Please add an extra line between Section comment and the strings. Check the examples provided here: https://github.com/mozilla-l10n/documentation/blob/main/src/localization/dev_best_practices.md#fluent-files